.gitignore

```diff
@@ -1,6 +1,8 @@
-CMakeFiles
-Makefile
-CMakeCache.txt
+# build artifacts
+build/
+CMakeFiles/
+Makefile/
+CMakeCache.txt/
```

Comment:
Are these every directories?  I think they are only files, so the trailing `/` will cause them to not be ignored.

Reply:
pfft. I have no idea why I did that. I'll fix it.